lib/data/data_sources/remote/api_const.dart

```diff
@@ -0,0 +1,20 @@
+const status_code_ok = 200;
+
+const image_assets_folder = 'assets/images';
+
+//region API Routes
+const api_base_url = 'https://api.themoviedb.org/3';
+const api_key = 'api_key=0d04409c636fcb41c638d670b3ca7583';
+const api_movie_path = 'movie';
+const api_genre_path = 'genre';
+const api_list_path = 'list';
+const api_similar_path = 'similar';
+const api_now_playing_path = 'now_playing';
+const api_upcoming_path = 'upcoming';
+const api_recommendations_path = 'recommendations';
+const api_popular_movies_path = 'popular';
+const api_top_rated_path = 'top_rated';
+const api_latest_movie_path = 'latest';
+const api_images_url = 'https://image.tmdb.org/t/p/original';
+const api_successful_response_code = 200;
```

Comment:
Instead to use this const you can use HttpStatus.OK

Reply:
done